chef/cookbooks/repos/recipes/default.rb

```diff
@@ -81,10 +89,11 @@
     end
   end
 
-  if node[:platform_family] != "suse" && node[:platform_family] != "windows"
-    template "/etc/gemrc" do
-      variables(admin_ip: address, web_port: web_port)
-      mode "0644"
-    end
-  end
+# FIXME: kill rubygem mirror?
+#  if node[:platform_family] != "suse" && node[:platform_family] != "windows"
+#    template "/etc/gemrc" do
+#      variables(admin_ip: address, web_port: web_port)
+#      mode "0644"
+#    end
+#  end
```

Comment:
Style/CommentIndentation: Incorrect indentation detected (column 0 instead of 2).